ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Config.scala

```diff
@@ -92,249 +92,280 @@ object Config {
       args: collection.Seq[String],
       getEnvVar: String => Option[String] = sys.env.get(_),
   ): Option[Config[Extra]] =
-    parser(name, extraOptions, getEnvVar).parse(args, createDefault(defaultExtra)).flatMap {
```

Comment:
Please review this file with the "hide whitespace" option enabled. for some reason `fmt.sh` decided to reformat the entire file.

Reply:
If you join the next two lines and then reformat, it should go back to normal.

`scalafmt` is sometimes not opinionated enough. If you split method calls onto multiple lines, it will honor that, even if they would fit on one line.

Reply:
Thanks for the tip.